en/mkdocs.yml

```diff
@@ -94,6 +94,7 @@ nav:
     - 'Account Information Service Flow': try-out/account-information-service-flow.md
     - 'Payment Initiation Service Flow': try-out/payment-initiation-service-flow.md
     - 'Confirmation of Funds flow': try-out/confirmation-of-funds-flow.md
+    - 'Deploy Custom Throttling Policies': try-out/custom-throttling-policies.md
```

Comment:
Shall we add this under "Learn Section" cause under try out flow we are discussing about main API flows. Learn section is for all feature explanations.